### PR TITLE
feat: add VcIssuerMock.json (a testable mock issuer implementation)

### PR DESCRIPTION
**Pull request by laysakura**
Already tested that all vc2.0 tests passed.

The test target architecture and deployment settings are documented in: https://github.com/laysakura/vc-issuer-mock/blob/main/crates/vc-issuer-mock-core/tests-vc-api/README.md

### CODE REVIEW

**Comment by aljones15** [comment]
this cool thanks

File: implementations/VcIssuerMock.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "VC Issuer Mock",
+  "implementation": "vc-issuer-mock-core",
+  "issuers": [{
+      "id": "https://github.com/laysakura/vc-issuer-mock",
```
Comment (aljones15) [inline implementations/VcIssuerMock.json:5]: so the issuer id is used the `issuer` value on the actual VC so it needs to match the `did:key` that your issuer signs with.

**Review by aljones15 — CHANGES_REQUESTED** [review]
if possible the issuer should sign with a did key and hence the `issuer.id` should be that did key.

File: implementations/VcIssuerMock.json (hunk 1 above)
Comment (laysakura) [reply implementations/VcIssuerMock.json]: @aljones15 Did it in https://github.com/w3c/vc-test-suite-implementations/pull/53/commits/e669feadd96aadba33956ab366469dc9778c0f27

**Comment by aljones15** [comment]
@laysakura I like your implementation, but you list it as a mock implementation. How is this a mock? As it is currently tagged it would end up the report here: https://w3c.github.io/vc-data-model-2.0-test-suite/

Are you ok with your implementation showing up in that report? Maybe you should mark it as a beta or an alpha if your implementation does actually really sign and verify 2.0 VCs.

**Comment by laysakura** [comment]
@aljones15 Thank you for your feedback.

To clarify, this implementation is designed to provide a VC-API for Issuers that can be used in unit tests by those working with VCs (such as Issuer Coordinators or Holder Coordinators in VC-API terminology). It does actually sign VCs in accordance with VCDM v2.0 specifications.

I agree that labeling it as an alpha implementation would be more appropriate, and I will update it accordingly.

With these clarifications, I would be comfortable with this implementation appearing in the report you mentioned. If there are no further concerns, I hope it can be included.

**Comment by aljones15** [comment]
@laysakura More than happy to have another implementation so feel free to merge, but do keep in mind mock implies a fake issuer, which might create misunderstandings about what is mocked with your issuer and verifier.

**Comment by BigBlueHat** [comment]
Thank you for building this, @laysakura! I think it's a great addition to the test suite results and a huge asset for folks wanting to wrap their implementations in VC-API for use with the test suites (or beyond).

In addition to adding it here, I hope to also promote it to other implementers who need this sort of wrapping approach--which should further lower the bar for implementers to get added to the test suite.

Much appreciated, @laysakura! 🙇

**Pull request merged**